lib/session.js

```diff
@@ -30,28 +30,28 @@ function hash(str) {
 
```

Comment:
All changes in this are unrelated to this PR.  They are good changes, but unrelated.  It'd be sad if something stopped this PR and these changes didn't make it in.  Please make another PR with just these and tag me so we can merge this.